src/containers/bookmarked-offers/BookmarksToolbar.styles.tsx

```diff
@@ -8,6 +8,7 @@ export const styles = {
     alignItems: 'center',
     gap: '10px',
     scrollMarginTop: '16px',
+    marginBottom: '1rem',
```

Comment:
Why are you using `rem` here? How this margin is related to document font size?

Reply:
I will change `rem` to `px`.